test/Hpack/ConfigSpec.hs

```diff
@@ -258,6 +258,15 @@ spec = do
         )
         (`shouldBe` package {packageLicenseFile = Just "LICENSE"})
 
+    it "accepts license file" $ do
+      withPackageConfig [i|
+        license-file: FOO
+        |]
+        (do
+        touch "FOO"
```

Comment:
In cases where the user specifies the license file explicitly I would always include it, without any existence checks.